secret_santa.py

```diff
@@ -105,30 +106,30 @@ def main(argv=None):
         participants = config['PARTICIPANTS']
         if len(participants) < 2:
             raise Exception('Not enough participants specified.')
-        
+
         givers = []
         for person in participants:
-            name, email, amazon = re.match(r'([^<]*)<([^>]*)>.*(http.*)', person).groups()
+            name, email, amazon, country = re.match(r'([^<]*)<([^>]*)>.*(http.*) ([A-z]{2})', person).groups()
             name = name.strip()
```

Comment:
I don't think this is quite right.  The idea is that the user would enter the country name and we could match it to  a region.  ie.  transformations.cn_to_ctn('Italy')  ==> Europe